spec/lib/bamboozled/api/employee_spec.rb

```diff
@@ -39,57 +52,56 @@
 
     info[:table][:row].first[:field].each do |f|
       case f[:id]
-      when 'location'
+      when "location"
         expect(f[:__content__]).to eq "New York Office"
-      when 'division'
+      when "division"
         expect(f[:__content__]).to eq "Sprockets"
-      when 'department'
+      when "department"
         expect(f[:__content__]).to eq "Research and Development"
-      when 'jobTitle'
+      when "jobTitle"
         expect(f[:__content__]).to eq "Machinist"
-      when 'reportsTo'
+      when "reportsTo"
         expect(f[:__content__]).to eq "John Smith"
       end
     end
   end
 
   it "Gets an employee's time off estimate" do
-    response = File.new('spec/fixtures/time_off_estimate.json')
+    response = File.new("spec/fixtures/time_off_estimate.json")
     stub_request(:any, /.*api\.bamboohr\.com.*/).to_return(response)
 
     future = Time.now + (60 * 60 * 24 * 180)
     estimate = @client.employee.time_off_estimate(1234, future)
 
-
     expect(estimate).to be_a Hash
-    expect(estimate["estimates"].keys).to match_array ["end", "estimate"]
+    expect(estimate["estimates"].keys).to match_array %w(end estimate)
     expect(estimate["estimates"]["estimate"].count).to eq 2
-    expect(estimate["estimates"]["estimate"].first.keys).to match_array ["timeOffType", "name", "units", "balance"]
+    expect(estimate["estimates"]["estimate"].first.keys)
+      .to match_array %w(timeOffType name units balance)
```

Comment:
This is a default Hound rule that will be overwritten with our custom `rubocop` rules.